cmake/private/FairRootTargets.cmake

```
@@ -36,3 +36,13 @@ function(fairroot_library_settings target)
   add_library(FairRoot::${target} ALIAS ${target})
   set_target_properties(${target} PROPERTIES ${PROJECT_LIBRARY_PROPERTIES})
 endfunction()
+
+
+set(PROJECT_EXPORT_SET FairRootTargets)
+
+
+function(fairroot_install_exported_library target)
+  install(TARGETS ${target}
+    EXPORT ${PROJECT_EXPORT_SET}
+    LIBRARY DESTINATION ${PROJECT_INSTALL_LIBDIR})
+endfunction()
```

```suggestion
function(fairroot_install)
  cmake_parse_arguments(PARSE_ARGV 0 ARGS "" "" "TARGETS")

  install(TARGETS ${ARGS_TARGETS}
    EXPORT ${PROJECT_EXPORT_SET}
    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
    ARCHIVE ${PROJECT_INSTALL_LIBDIR}
    LIBRARY DESTINATION ${PROJECT_INSTALL_LIBDIR})
endfunction()
```

* doesn't need to be library-specific

---

Updated, force pushed.
I named it still `fairroot_install_exported`, because it does the EXPORT unconditionally for now.
As it is a private function/command, we can rename it as needed.